bcp47/library/Data/BCP47/Internal/Parser.hs

```diff
@@ -1,12 +1,16 @@
 module Data.BCP47.Internal.Parser
   ( complete
+  , asciiLetterDigit
+  , asciiLetter
+  , asciiDigit
```

Comment:
Do these parser changes solve the other issues pointed out in #21? (tags `es-419` and `cmn`)

Reply:
Lemme check, those might still be blocked by the libraries we use.

Reply:
`en-gb` is solved, but `es-419` and `en-TP` don't work due to restrictions in the `country` library we use. `cmn` doesn't work because we don't support lone ext-lang tags yet (we expect every tag always starst with an ISO-639-1 code). I added pending tests with explanations [here](https://github.com/freckle/bcp47/pull/26/files#diff-99440716d53af3a64d1960d04139dfa9d69c004f9efeed0cb1806029fde94e4dR71).